how_to_x.md

```diff
@@ -34,4 +34,14 @@ Optionally pass `--resume` for unversioned builds.
 And finally build without prebuild, but with postbuild
 ```
 makelove -d prebuild -v 1.2.3 appimage (--resume)
-```
\ No newline at end of file
+```
+
+## Build windows targets on macOS
+
+Install wine (e.g., homebrew, shown here):
+
+```bash
+brew cask install wine-stable
+```
+
+Then build windows targets as normal (and run them!)
```

Comment:
I think the README is already a little too big. I'd much prefer this suggestion to be printed if Wine can not be found on the system.